src/elements/content-explorer/ItemGrid.js

```diff
@@ -22,20 +42,51 @@ const ItemGrid = ({ currentCollection, gridColumnCount, rootId, ...rest }: Props
      */
     const slotRenderer = (slotIndex: number): ?React.Element<any> => {
         const item: ?BoxItem = getProp(currentCollection, `items[${slotIndex}]`);
+        return item ? <ItemGridCell item={item} onItemSelect={onItemSelect} rootId={rootId} {...rest} /> : null;
+    };
+
+    /**
+     * Update the currently selected item when navigating with keyboard and mouse
```

Comment:
is the `and mouse` part accurate? I thought the ArrowKeyStepper is for the arrow keys